src/classes/TDTM_DefaultConfig.cls

```diff
@@ -184,7 +184,11 @@ public without sharing class TDTM_DefaultConfig {
               Class__c = 'RD_RecurringDonations_TDTM', Load_Order__c = 1, Object__c = 'npe03__Recurring_Donation__c',
               Trigger_Action__c = 'BeforeDelete;BeforeInsert;BeforeUpdate;AfterInsert;AfterUpdate'));
 
-        // Enhanced Recurring Donations trigger handler
+        // Enhanced Recurring Donations trigger handlers
+        handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
```

Comment:
@force2b , @rponti-sforg : new trigger handler is added to propagate changes from Opps to the related Recurring Donation.

Reply:
@rponti-sforg , need to reload the NPSP Settings trigger handlers before testing so the most recent configuration is reloaded.